crates/sui/src/keytool.rs

```diff
@@ -339,36 +339,37 @@ impl KeyToolCommand {
             }
 
             KeyToolCommand::Import {
-                mnemonic_phrase,
+                input_string,
                 key_scheme,
                 derivation_path,
             } => {
-                let address =
-                    keystore.import_from_mnemonic(&mnemonic_phrase, key_scheme, derivation_path)?;
-                info!("Key imported for address [{address}]");
+                // check if it is a private key or a mnemonic
+                if input_string.starts_with("0x") && input_string.len() == 66 {
```

Comment:
How should we interpret the `key_scheme` and `derivation_path` inputs in this case?

Reply:
Currently, from discussions with Jordan in principle we only support ED25519 in the wallet. Thus, I decided to only allow `ed25519` as a key scheme when calling the `keytool import` command and providing a private key.